ci-operator/step-registry/upi/install/aws/cluster/upi-install-aws-cluster-commands.sh

```diff
@@ -71,7 +70,18 @@ export BOOTSTRAP_URI
 
 # begin bootstrapping
 if openshift-install coreos print-stream-json 2>/tmp/err.txt >coreos.json; then
-  RHCOS_AMI="$(jq -r --arg region "$AWS_REGION" '.architectures.x86_64.images.aws.regions[$region].image' coreos.json)"
+  if [[ "${CLUSTER_TYPE}" == "aws-arm64" ]]; then
+    RHCOS_AMI="$(jq -r --arg region "$AWS_REGION" '.architectures.aarch64.images.aws.regions[$region].image' coreos.json)"
+    BOOTSTRAP_INSTANCE_TYPE=m6g.large
+    MASTER_INSTANCE_TYPE=m6g.xlarge
+    WORKER_INSTANCE_TYPE=m6g.xlarge
+    CLUSTER_TYPE=aws 
+  else
+    RHCOS_AMI="$(jq -r --arg region "$AWS_REGION" '.architectures.x86_64.images.aws.regions[$region].image' coreos.json)"
+    BOOTSTRAP_INSTANCE_TYPE=m5.large
+    MASTER_INSTANCE_TYPE=m5.xlarge
+    WORKER_INSTANCE_TYPE=m5.xlarge 
```

Comment:
do we not want it to be the same as m4.xlarge below ? i  mean no harm in upgrading though